src/styles/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  background: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  align-self: center;
+  object-fit: contain;
+  object-position: center;
+}
+
+.card__info {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--grey);
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.card__review {
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: flex-end;
+  font-weight: 400;
+  text-align: right;
+  color: var(--dark);
+}
+
+.card__price-text {
+  display: flex;
+  justify-content: space-between;
+  font-weight: 400;
+  margin-bottom: 16px;
+}
+  
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--grey);
+}
+
+.card__price-sum {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--dark);
+}
+
+.card__button {
+  display: flex;
+  text-decoration: none;
+  background: var(--light-blue);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--white);
+  padding: 12px 70px;
```

Comment:
why do we need to add paddings to the button? just add height, fullwidth, and center text with `flex`